rtc-video-quality/generate_data.py

```diff
@@ -349,9 +349,11 @@ def run_command(job, encoder_command, job_temp_dir, encoded_file_dir):
     clip = job['clip']
     start_time = time.time()
     try:
-        process = subprocess.Popen(command,
+        process = subprocess.Popen(' '.join(command),
```

Comment:
This won't work correctly if any of the words in the command contain spaces, or need to be quoted for any other reason. In Python 3.8, you could use shlex.join. If you need to run in an earlier version of Python 3, it's just
' '.join(shlex.quote(arg) for arg in command)

Reply:
I didn't know about this. We're currently using Python 3.7 but I'll update this

Reply:
[`shlex.quote`](https://docs.python.org/3/library/shlex.html#shlex.quote) returns an shell-escaped version of a string but I need `&&` in my command.  

I adjusted your call a bit by writing `' '.join(shlex.quote(arg) if arg != '&&' else arg for arg in command)` instead

Reply:
Oh, good point.